tests/conftest.py

```diff
@@ -22,7 +22,7 @@ def create_tiff(path: Path, n_files: int):
 
 
 @pytest.fixture
-def minimum_algorithm_custom() -> dict:
+def minimum_algorithm_n2v() -> dict:
```

Comment:
The git comparison has got a bit messed up, but has custom algorithm been removed?

Reply:
Looking into the code, it seems that the rational for `custom` algorithm was to make sure to escape any assumption on the model parameters: https://github.com/CAREamics/careamics/blob/aed4d82387590159a005f3e8d6e08fc8efad168f/src/careamics/config/configuration_model.py#L219.

That means that it should not be removed!

Reply:
I added back the `custom` algorithm, and added tests that were missing to test what it was actually used for.